fetch.bs

```diff
@@ -4978,6 +5004,29 @@ steps:
       <p class=note>The `<code>GET</code>` <a for=/>method</a> restriction serves no useful purpose
       other than being interoperable.
 
+     <li>
+      <p>Let <var>blobStorageKey</var> be the result of running
+      <a>obtain a storage key for non-storage purposes</a> with <var>blobURLEntry</var>'s
+      <a>environment settings object</a>.
```

Comment:
This should not link to the type definition of "environment settings object", but instead link to https://w3c.github.io/FileAPI/#blob-url-entry-environment .

Reply:
ah yes thanks for pointing this out. Changed